### PR TITLE
Test Integration Fixes: Enhancing Stability

### PR DESCRIPTION
**Pull request by george-ghawali**
- Adding some delay after creating static route since it needs some time to be active
- Add minor fix in service groups test that assert empty list

### CODE REVIEW

File: tests/integration/targets/ntnx_static_routes/tasks/create.yml [hunk 1]
```diff
@@ -89,12 +119,32 @@
       - result.response is defined
       - result.response.status.state == 'COMPLETE'
       - result.changed == true
-      - result.response.status.resources.static_routes_list[0]["is_active"] == true
       - result.response.status.resources.static_routes_list[0]["destination"] == "10.2.4.0/24"
       - result.response.status.resources.static_routes_list[0]["nexthop"]["external_subnet_reference"]["name"] == "{{ external_nat_subnet.name }}"
     fail_msg: "Static routes overriding failed"
     success_msg: "Static routes overridden successfully"
 
+- name: Wait 1 minute for static routes to be created
```
Comment (bhati-pradeep) [inline tests/integration/targets/ntnx_static_routes/tasks/create.yml:127]:
```suggestion
- name: Wait 1 minute for static routes to become active 
```

Comment (bhati-pradeep) [reply tests/integration/targets/ntnx_static_routes/tasks/create.yml:127]: Update this everywhere

**Review by bhati-pradeep — CHANGES_REQUESTED** [review]

File: tests/integration/targets/ntnx_static_routes/tasks/create.yml [hunk 2]
```diff
@@ -30,6 +30,36 @@
...
       - result.response.status.state == 'COMPLETE'
       - result.changed == true
+      - ("{{d1}}" == "10.2.2.0/24" and "{{d2}}" == "10.2.3.0/24") or ("{{d2}}" == "10.2.2.0/24" and "{{d1}}" == "10.2.3.0/24")
+      - result.response.status.resources.static_routes_list[0]["nexthop"]["external_subnet_reference"]["name"] == "{{ external_nat_subnet.name }}"
+      - result.response.status.resources.static_routes_list[1]["nexthop"]["external_subnet_reference"]["name"] == "{{ external_nat_subnet.name }}"
+      - result.response.status.resources.default_route["destination"] == "0.0.0.0/0"
+      - result.response.status.resources.default_route["nexthop"]["external_subnet_reference"]["name"] == "{{ external_nat_subnet.name }}"
+
+    fail_msg: "Fail: Unable to update static routes of vpc"
+    success_msg: "Succes: static routes updated successfully"
```
Comment (abhinavbansal29) [inline tests/integration/targets/ntnx_static_routes/tasks/create.yml:40]:
```suggestion
    success_msg: "Success: static routes updated successfully"
```

File: tests/integration/targets/ntnx_static_routes/tasks/create.yml [hunk 3]
```diff
@@ -39,8 +69,8 @@
       - result.response.status.resources.default_route["destination"] == "0.0.0.0/0"
       - result.response.status.resources.default_route["nexthop"]["external_subnet_reference"]["name"] == "{{ external_nat_subnet.name }}"
 
-    fail_msg: 'Fail: Unable to update static routes of vpc'
-    success_msg: 'Succes: static routes updated successfully'
+    fail_msg: "Fail: Unable to get static routes for vpc"
+    success_msg: "Succes"
```
Comment (abhinavbansal29) [inline tests/integration/targets/ntnx_static_routes/tasks/create.yml:73]:
```suggestion
    success_msg: "Success"
```

**Review by abhinavbansal29 — CHANGES_REQUESTED** [review]

File: tests/integration/targets/ntnx_static_routes/tasks/create.yml (hunk 3 above)
Comment (george-ghawali) [reply tests/integration/targets/ntnx_static_routes/tasks/create.yml:73]: fixed in different PR
